.travis.yml

```diff
@@ -79,6 +83,7 @@ matrix:
         - sourceline: 'deb http://download.opensuse.org/repositories/network:/messaging:/zeromq:/git-draft/xUbuntu_14.04/ ./'
           key_url: 'http://download.opensuse.org/repositories/network:/messaging:/zeromq:/git-draft/xUbuntu_14.04/Release.key'
         packages:
+        - zproject
```

Comment:
This installs it only in the ASAN job, not sure it's what you intended?

I'd suggest to add a new -env stanza instead

Reply:
i... dunno? just trying to mirror what zyre/.travis had to make sure i didn't break something on someone. pulled it, re-running travis job see how it plays out.